content/en/developers/integrations/check_references.md

```diff
@@ -125,7 +125,7 @@ You can find the complete list of mandatory and optional attributes for the `met
 | `description`   | Optional           | Description of the metric.                                                                                                                                                                                                                                                                                                              |
 | `orientation`   | Mandatory          | Set to `1` if the metric should go up, such as `myapp.turnover`. Set to `0` if the metric variations are irrelevant. Set to `-1` if the metric should go down, such as `myapp.latency`.                                                                                                                                                         |
 | `integration`   | Mandatory          | The name of the integration that emits the metric. Must be the normalized version of the `tile.title` from the `manifest.json` file. Any character besides letters, underscores, dashes, and numbers are converted to underscores. For example: `Openstack Controller` -> `openstack_controller`, `ASP.NET` -> `asp_net`, and `CRI-o` -> `cri-o`. |
-| `short_name`    | Mandatory          | Explicit unique ID for the metric.                                                                                                                                                                                                                                                                                                      |
+| `short_name`    | Mandatory          | A human-readable, abbreviated version of the metric name. Do not repeat the integration name. For example: `postgresql.index_rows_read` -> `idx rows read`.                                                                                                                                                                                                                                                                                                     |
```

Comment:
```suggestion
| `short_name`    | Mandatory          | A human-readable, abbreviated version of the metric name. Do not repeat the integration name. For example, instead of `postgresql.index_rows_read`, use `idx rows read`.                                                                                                                                                                                                                                                                                                     |
```